pegasus/forms/class_submission.rb

```diff
@@ -148,6 +148,32 @@ def self.process_with_ip(data, created_ip)
     end
   end
 
+  # This is almost identical to the below function, but it's called by
```

Comment:
Is the idea that this duplicated code will only be temporary? If so, we should make a note of which method will be deleted once Solr is removed, if not we should refactor the methods to avoid duplication.

Reply:
Yes, I thought about refactoring but decided we could just delete the lower function when the Solr deprecation is done.